homeassistant/components/openhardwaremonitor/sensor.py

```diff
@@ -89,12 +94,16 @@ def update(self):
             values = array[path_number]
 
             if path_index == len(self.path) - 1:
-                self.value = values[OHM_VALUE].split(" ")[0]
+                self.value = self.parse_number(values[OHM_VALUE].split(" ")[0])
```

Comment:
`self.value` is now a float and returned in the `state` property method.
However, states in Home Assistant are always stored as a string. Why convert it to a float first, while HA will convert it back to a string again?

Reply:
Hello!

Sorry, I did not know that it just converts it back to a string.

I have fixed this issue by removing the float cast and I added an extra test to show what a value can look like on a different locale.